### PR TITLE
Implement sign_in

### PR DESCRIPTION
**Pull request by kolyanjy**

### CODE REVIEW

File: spec/requests/graphql/mutations/user/sign_in_spec.rb [hunk 1]
```diff
@@ -16,21 +16,27 @@
...
       expect(response.status).to be(200)
     end
   end
 
   context 'when wrong credendials' do
-    let(:buyer_account) { create(:buyer_account, status: 1) }
+    let(:user_account) { create(:user_account) }
+    let(:default_variables) do
+    {
+      email: 'lol@gmail.com',
```
Comment (yaorlov) [inline spec/requests/graphql/mutations/user/sign_in_spec.rb]: let's rename it

File: spec/support/graphql/mutations_helpers/user.rb [hunk 2]
```diff
@@ -13,5 +13,23 @@ def user_signup_mutation
         }
       )
     end
+
+    def user_signin_mutation
+      %(
+        mutation userSignIn(
+          $email: String!,
```
Comment (yaorlov) [inline spec/support/graphql/mutations_helpers/user.rb]: let's use InputType here

File: spec/support/schemas/authentication_account_error_schema.rb [hunk 3]
```diff
@@ -0,0 +1,16 @@
+AuthenticationAccountErrorSchema = Dry::Validation.Schema do
```
Comment (yaorlov) [inline spec/support/schemas/authentication_account_error_schema.rb]: check with rubocop

Comment (yaorlov) [reply spec/support/schemas/authentication_account_error_schema.rb]: maybe rename it to `UnauthenticatedErrorSchema`?

File: app/concepts/user_auth/operation/sign_in.rb [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+    step :check_credentials
+    step :generate_auth_token
+
+    def build_contract(_ctx, params:, **)
+      UserAuth::Contract::SignIn.new(UserAccount, **params)
+    end
+    
+    def find_user_account(ctx, params:, **)
+      ctx['user_account'] = UserAccount.find_by(email: params[:email])
+      Trailblazer::Activity::Right
```
Comment (bl0rch1d) [inline app/concepts/user_auth/operation/sign_in.rb]: you can make this step as `pass :find_user_account`

Comment (bl0rch1d) [inline app/concepts/user_auth/operation/sign_in.rb]: Why right?

Comment (kolyanjy) [reply app/concepts/user_auth/operation/sign_in.rb]: Because according to tibetan drawings ☯️

File: schema.graphql [hunk 5]
```diff
@@ -19,6 +19,11 @@ type AuthTokenType {
 }
 
 type Mutation {
+  """
+  translation missing: en.graphql.mutations.user.sign_in.desc
```
Comment (yaorlov) [inline schema.graphql]: please add translations

**Pull request merged**